.github/workflows/development.yml

```diff
@@ -0,0 +1,50 @@
+name: Development
+on:
+  pull_request:
```

Comment:
i think worth adding "on `workflow_dispatch`", at least to this one, but maybe to both? (i think it can be useful for basically every GHA workflow to have this)
https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow